src/NuGet.Core/NuGet.Configuration/Proxy/ProxyCache.cs

```diff
@@ -99,6 +99,14 @@ public WebProxy GetUserConfiguredProxy()
                 {
                     webProxy.Credentials = new NetworkCredential(userName, password);
                 }
+                
+                var noProxy = _settings.GetValue(SettingsUtility.ConfigSection, ConfigurationContants.NoProxy);
```

Comment:
Not sure if I follow you here. The "no_proxy" statements are not identical, since the 1st block reads from `_settings` and the 2nd from `_environment`. The only duplication is the ','-based splitting lines

I don't know the NuGet documentation system myself, so I don't know how or where to document "no_proxy". Would it be possible for you to fix that in a later PR?